src/ert/storage/local_ensemble.py

```diff
@@ -545,6 +550,39 @@ def _load_parameters_lazy(
         df = pl.scan_parquet(group_path)
         return df
 
+    def _load_scalar_keys(
+        self,
+        keys: list[str],
+        realizations: int | npt.NDArray[np.int_] | None = None,
+        transformed: bool = False,
+    ) -> pl.DataFrame:
+        df_lazy = self._load_parameters_lazy(GenKwConfig.scalar_file())
+        df_lazy = df_lazy.select(["realization", *keys])
+        if realizations is not None:
+            if isinstance(realizations, int):
+                realizations = np.array([realizations])
+            df_lazy = df_lazy.filter(pl.col("realization").is_in(realizations))
+        df = df_lazy.collect(engine="streaming")
```

Comment:
Do we need lazy loading and streaming for scalar parameters? They are usually in the 10s and 100s so do not use much memory.

Reply:
no need. I will get rid of the lazy load :+1: